spec/models/custom_link_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe CustomLink, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
please add tests before merging

Reply:
Yes sure... It is still a work in progress